components/skeletons/NotificationSkeleton/NotificationSkeleton.type.ts

```diff
@@ -0,0 +1,6 @@
+type TSize = 'small' | 'medium' | 'large';
```

Comment:
you removed size classes but why did not remove this?

Reply:
I apoligize to forget this place